src/v2/Apps/Artwork/Components/ArtworkVideoPlayer/ArtworkVideoPlayer.tsx

```diff
@@ -35,9 +36,10 @@ const ArtworkVideoPlayer: FC<ArtworkVideoPlayerProps> = ({
       alignItems="center"
       justifyContent="center"
     >
-      <ResponsiveBox
+      <Casette
         bg="black10"
-        mx={[0, 2]}
+        mx={[0, 4]}
+        paddingX={[0, 2]}
```

Comment:
totally #superminor, but can do `px` as we do elsewhere!

Reply:
shhhhh 🙈 🙉 🙊

Reply:
fixed